revenuecat_examples/purchase_tester/lib/src/app.dart

```diff
@@ -3,6 +3,7 @@ import 'dart:async';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:purchases_flutter/purchases_flutter.dart';
+import 'package:purchases_flutter_ui/purchases_flutter_ui.dart';
```

Comment:
This is the new import, but the class is called `RevenueCatUI`. I thought it was ok, but lmk if you would prefer to rename the package as well.